igvc_navigation/src/trajectory_planner/motion_profiler.h

```diff
@@ -0,0 +1,46 @@
+#ifndef SRC_MOTION_PROFILER_H
+#define SRC_MOTION_PROFILER_H
+
+#include <igvc_msgs/trajectory.h>
+#include <igvc_utils/robot_state.h>
+
+struct WheelConstraint
+{
+  double velocity;
+  double acceleration;
+};
+
+struct RobotConstraint
+{
+  double velocity;
+  double acceleration;
+};
+
+struct MotionProfilerOptions
+{
+  double beta;
+  double lambda;
```

Comment:
These are not useful names. Please mention what they actually do, or, even better, change them to be actually descriptive. If you're going to keep them as beta/lambda, please reference some paper or other source which explains what they are in more detail (but prefer renaming, because it's a lot less overhead).

Reply:
yeah, they're in the Smooth Control paper. I'll link the paper in the comments.